index.js

```diff
@@ -152,10 +221,12 @@ function parseArch(opts) {
     opts.arch = 'i686';
   } else if (opts.bits === '32' && opts.platform === 'win32') {
     opts.arch = 'i386';
-  } else {
-    opts.arch = 'x86_64';
-  }
-
+  } else if (opts.arch === 's390x' || opts.arch === 'ppc64le') {
+    opts.enterprise = true;
+  } else if (opts.arch.match('arm')) {
+    // ARM can be Enterprise or community, how do we decide?
```

Comment:
imho it should be determined based on whether `enterprise === true`, and `enterprise` should be marked `true` IFF `evergreen` is chosen

Reply:
Does that mean we'll only test Enterprise builds on Evergreen?

Reply:
yes I think that's a fine assumption